### ISSUE TITLE
0001078:
email features

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#628]
**Reported by pschuele on 12 Jun 2009 11:03**

nice to have features:
see children

**Comment by Gloirin** [comment]
Depends on #832

**Comment by Gloirin** [comment]
Depends on #874

**Comment by Gloirin** [comment]
Depends on #871

**Comment by Gloirin** [comment]
Depends on #742

**Comment by Gloirin** [comment]
Depends on #795

**Comment by Gloirin** [comment]
Depends on #796

**Comment by Gloirin** [comment]
Depends on #813

**Comment by Gloirin** [comment]
Depends on #885

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 1 Sep 2009 12:09**

resolved all children
